arel_toolkit.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'github_changelog_generator', '~> 1.14.3'
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec', '~> 3.8'
+  spec.add_development_dependency 'rspec-rails', '~> 3.8'
```

Comment:
So adding this gem will cause Rails to be included as well?

Reply:
Looking at the lock file, it includes 
```
      actionpack (>= 3.0)
      activesupport (>= 3.0)
```